LICENSE-MIT-0

```diff
@@ -0,0 +1,19 @@
+MIT No Attribution (MIT-0)
+
+Copyright (c) 2022 Karolina Surma
```

Comment:
I couldn't find any relevant sources about how to properly relicense the part of project given there are more than one contributors, so I just took the header from the MIT license in the same repository. Do you know how to do it properly? I'll be happy to add your name here too, @hroncok.

Reply:
I'd use the same copyright as stated in https://github.com/befeleme/pyp2spec/blob/main/LICENSE-MIT